src/background/api/request.js

```diff
@@ -36,3 +36,8 @@ export function getOasisClient(){
   const oasisClient = new oasis.client.NodeInternal(fetchUrl)
   return oasisClient
 }
+
+// to make commonFetch jest.spyOn'able
+export default {
+  commonFetch
+}
```

Comment:
I think you can `jest.mock('./request')` to spy on commonFetch without this

Reply:
Fixed!